### PR TITLE
Dark theme

### PR DESCRIPTION
**Pull request by Mini-Ware**

### CODE REVIEW

File: api/index.js [hunk 1]
```diff
@@ -3,8 +3,9 @@ const renderSVG = require('../src/renderer/renderSVG')
...
     const {type} = req.query
+    const {theme} = req.query
     var data = await fetchQuotes()
     res.setHeader("Content-Type", "image/svg+xml")
     res.setHeader("Cache-Control", `public, max-age=600`);
-    res.send(renderSVG(data, type))
-}
\ No newline at end of file
+    res.send(renderSVG(data, type, theme))
+})
```
Comment (PiyushSuthar) [inline api/index.js:11]: The parenthesis isn't required here, it's causing an error.

Comment (PiyushSuthar) [reply api/index.js]: nvm, fixed

**Comment by PiyushSuthar** [comment]
Looks good to me! Thanks for the PR!!!

File: api/index.js (hunk 1 above)
Comment (Mini-Ware) [reply api/index.js]: oh okay, thanks for letting me know

**Pull request merged**